src/_layouts/default.html

```diff
@@ -8,7 +8,7 @@
       {% include navigation-toc.html %}
       <article>
         <div class="content">
-          {% include banner.html %}
+          {% comment %} {% include banner.html %} {% endcomment %}
```

Comment:
Adding / removing the banner is common enough. We could control its inclusion via a site config field (something like:

```yaml
show_banner: true # or false or simply excluded
```

Then in this file:

```
{% if site.show_banner %} {% include banner.html %} {% endif %}
```

(with a similar change in `src/index.hmlt`.